assets/js/common/session.js

```diff
@@ -69,15 +72,23 @@ function handleSessionExpiration() {
 }
 
 function getUserSession() {
-    return $.ajax({
+    const sessionCheck = $.ajax({
```

Comment:
Probably don't need the intermediary variable. Could be `return $.ajax({}).then(response => {…})`

Reply:
I tried this first and the other uses of `getUserSession()` stopped working... not sure jQuery returns the right thing if passed like that?

Reply:
That's surprising, but 👌

Reply:
Ah wait it's because I didn't `return response` inside the first `then()` call. Will add!